docs/network-spec/miniprotocols.tex

```diff
@@ -1494,21 +1494,10 @@ \section{Node-to-node protocol}
   \begin{center}
     \begin{tabular}{l|l}
       version         & description \\\hline\hline
-      $NodeToNodeV\_1*$ & initial version \\\hline
-      $NodeToNodeV\_2*$ & block size hints \\\hline
-      $NodeToNodeV\_3*$ & introduction of keep-alive mini-protocol \\\hline
-      $NodeToNodeV\_4*$ & introduction of diffusion mode in handshake mini-protocol \\\hline
-      $NodeToNodeV\_5*$ & \\\hline
-      $NodeToNodeV\_6*$ & transaction submission version 2 \\\hline
-      $NodeToNodeV\_7$ & new keep-alive, Alonzo ledger era \\\hline
-      $NodeToNodeV\_8$ & chain-sync \& block-fetch pipelining \\\hline
-      $NodeToNodeV\_9$ & Babbage ledger era \\\hline
-      $NodeToNodeV\_10$ & Full duplex connections \\\hline
-      $NodeToNodeV\_11$ & Peer sharing willingness \\\hline
-      $NodeToNodeV\_12$ & No observable changes \\\hline
```

Comment:
We should either move them to a separate section or record the very early versions in the code (this list is more complete than what we have in the code).  The former looks better to me.

Reply:
I've moved them to a new appendix chapter.